roles/translator/src/downstream_sv1/downstream.rs

```diff
@@ -447,7 +453,11 @@ impl IsServer<'static> for Downstream {
             let mut downstream_part: Vec<u8> = request.extra_nonce2.clone().into();
             tproxy_part.append(&mut downstream_part);
 
-            let to_send = (request.clone(), tproxy_part);
+            let to_send = SubmitShareWithChannelId {
+                channel_id: self.connection_id,
+                share: request.clone(),
+                extranonce: tproxy_part,
+            };
```

Comment:
While we are in this file can you fix these warnings (line 540) - i see them on so many PRs.